privaterelay/settings.py

```diff
@@ -569,6 +569,7 @@ def _get_initial_middleware():
 FXA_SETTINGS_URL = config('FXA_SETTINGS_URL', f'{FXA_BASE_ORIGIN}/settings')
 FXA_SUBSCRIPTIONS_URL = config('FXA_SUBSCRIPTIONS_URL', f'{FXA_BASE_ORIGIN}/subscriptions')
 FXA_SUPPORT_URL = config('FXA_SUPPORT_URL', f'{FXA_BASE_ORIGIN}/support/')
+BASKET_ORIGIN = config('BASKET_ORIGIN', 'https://basket.mozilla.org')
```

Comment:
Question: Should this be the dev URL value (`https://basket-dev.allizom.org`) or do we want to fallback to production here?

Reply:
I figured I'd rather risk accidentally sending test data to production, than sending production data to the dev environment, hence this choice.